spec/classes/jenkins_slave_spec.rb

```diff
@@ -67,6 +68,19 @@
     end
   end
 
+#  describe 'Darwin' do
```

Comment:
so the puppetlabs/java module doesn't support Darwin so I think part of why this is failing.